presto-parser/src/test/java/io/prestosql/sql/parser/TestSqlParser.java

```diff
@@ -1526,6 +1526,14 @@ public void testCommentTable()
         assertStatement("COMMENT ON TABLE a IS NULL", new Comment(Comment.Type.TABLE, QualifiedName.of("a"), Optional.empty()));
     }
 
+    @Test
+    public void testCommentColumn()
+    {
+        assertStatement("COMMENT ON COLUMN a.b IS 'test'", new Comment(Comment.Type.COLUMN, QualifiedName.of("a", "b"), Optional.of("test")));
```

Comment:
Add tests for other forms of column names: E.g., `catalog.schema.table.column`, `schema.table.column`, `column`, etc.